1.generate-profiles/profiling_config.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Per our recent conversation about the site_processing_config, should have at top:
```
---
master_config:
  step:generate-profiles
  pipeline: Pooled Cell Painting
```